packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -670,6 +670,9 @@ export default defineComponent({
 		},
 		// eslint-disable-next-line @typescript-eslint/no-explicit-any
 		onDataChange({ name, value }: { name: string; value: any }) {
+			// disregard default value being set on component mount
```

Comment:
Super nitpicking 😛, but the current comment is a bit confusing as the method doesn't know if it's being run on mount or in a different context. IMO, a more fitting comment could be `// Skip update if new value matches the current`

Reply:
Totally agree, updated!